### PR TITLE
AMP Meta area fixes/clean ups

### PR DESCRIPTION
**Pull request by zeftilldeath**
Added a "better meta" approach to AMP's meta area. 

# Before
screen shot 2017-01-11 at 11 19 51

# After
screen shot 2017-01-11 at 11 19 17

# Before
screen shot 2017-01-11 at 11 18 41

# After
screen shot 2017-01-11 at 11 18 56

# Expanded state
screen shot 2017-01-11 at 11 20 25

🎁  for you @ScottPainterGNM 

@NataliaLKB Please take a look at the is empty stuff, when you get a chance.

### CODE REVIEW

**Comment by ScottPainterGNM** [comment]
tumblr_miazecvm8q1r3ifxzo1_500

**Comment by NataliaLKB** [comment]
Looks great to me @zeftilldeath! I included one commit with some linting/cleanup.

I have asked @gtrufitt to review as well, because he worked on the PR for beta meta.

**Review by gtrufitt — APPROVED** [review]
A few points to fix-up but looks good! 

And I'm aware that I may have commented on some of the code copied from the original PR... 😕

File: static/src/stylesheets/amp/_social.scss [hunk 1]
```diff
@@ -31,6 +31,51 @@
...
+    float: none;
+    display: inline-block;
+    height: 32px;
+    width: 32px;
+}
+
+.social__item--revealer {
+    display: none;
+
+    &:checked {
```
Comment (gtrufitt) [inline static/src/stylesheets/amp/_social.scss]: I would maybe just seperate out `.social__item-revealer {}` and  `.social__item-revealer:checked` here as it reduced the complexity of that quite complex block!

File: static/src/stylesheets/amp/_content.scss [hunk 2]
```diff
@@ -189,16 +194,45 @@
...
+        height: 32px;
+        padding: 0;
+        border: 0;
+        background-color: transparent;
+        margin-right: 0;
+        white-space: nowrap;
+    }
+
+    & > a > div {
+        display: none;
```
Comment (gtrufitt) [inline static/src/stylesheets/amp/_content.scss]: Can you do this with an if statement in the template instead?

**Comment by zeftilldeath** [comment]
@gtrufitt Thanks for helping me out there, have managed to delete ALLLLLL of that css please take a look.

**Review by gtrufitt — APPROVED** [review]
Lgtm! (Sorry forgot this morning)

**Pull request merged**